CHANGELOG.md

```diff
@@ -2,8 +2,15 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## [19.1.0](https://github.com/ngneat/spectator/compare/v19.0.0...v19.1.0) (2024-09-12)
 
+* 🎸 Upgrade to Angular 19 ([7f3eb6c](https://github.com/ngneat/spectator/commit/7f3eb6c541fa9843a8e2f10ab8c77bca33a678c0))
```

Comment:
please revert this file

Reply:
OK - I thought that I was just supposed to revert the version header line, not all my Changelog edits.

As I recall, for my last contribution I added some Changelog documentation, so that's what I was doing here. I'll revert everything then.